### PR TITLE
[WIP][NNPI] Import BatchNorm instead of lowering

### PR DESCRIPTION
**Pull request by jfix71**
Summary: See title.

Differential Revision: D19594607

### CODE REVIEW

**Comment by jsubag** [comment]
@talc23 take a look at these changes.

**Comment by jfix71** [comment]
Done in https://github.com/pytorch/glow/pull/4167.
